docs/maps/tile-layer.asciidoc

```diff
@@ -0,0 +1,17 @@
+[[tile-layer]]
+== Tile layer
+
+Display image tiles served from a tile server.
+
+[role="screenshot"]
+image::maps/images/tile_layer.png[]
+
+Tile layers can be created from the following sources
+
+*Custom Tile Map Service*:: Map tiles configured in kibana.yml.
```

Comment:
possible to add link to `docs/setup/settings.asciidoc` page (?) and mention `map.tilemap.*` config

Reply:
<img width="760" alt="screen shot 2019-02-28 at 3 11 44 pm" src="https://user-images.githubusercontent.com/373691/53602276-832fd400-3b6b-11e9-811f-5b91e3261e3e.png">